public/app/features/logs/components/LogLabels.tsx

```diff
@@ -9,40 +9,79 @@ const HIDDEN_LABELS = ['level', 'lvl', 'filename'];
 
 interface Props {
   labels: Labels;
+  emptyMessage?: string;
 }
 
-export const LogLabels = ({ labels }: Props) => {
+export const LogLabels = ({ labels, emptyMessage }: Props) => {
   const styles = useStyles2(getStyles);
-  const displayLabels = Object.keys(labels).filter((label) => !label.startsWith('_') && !HIDDEN_LABELS.includes(label));
+  const displayLabels = useMemo(
+    () =>
+      Object.keys(labels)
+        .filter((label) => !label.startsWith('_') && !HIDDEN_LABELS.includes(label))
+        .sort(),
+    [labels]
+  );
 
-  if (displayLabels.length === 0) {
+  if (displayLabels.length === 0 && emptyMessage) {
     return (
       <span className={cx([styles.logsLabels])}>
-        <span className={cx([styles.logsLabel])}>(no unique labels)</span>
+        <span className={cx([styles.logsLabel])}>{emptyMessage}</span>
       </span>
     );
   }
 
   return (
     <span className={cx([styles.logsLabels])}>
-      {displayLabels.sort().map((label) => {
+      {displayLabels.map((label) => {
         const value = labels[label];
         if (!value) {
           return;
         }
-        const tooltip = `${label}: ${value}`;
+        const tooltip = `${label}=${value}`;
+        const labelValue = `${label}=${value}`;
         return (
-          <span key={label} className={cx([styles.logsLabel])}>
-            <span className={cx([styles.logsLabelValue])} title={tooltip}>
-              {value}
-            </span>
-          </span>
+          <LogLabel key={label} styles={styles} tooltip={tooltip}>
+            {labelValue}
+          </LogLabel>
         );
       })}
     </span>
   );
 };
 
+interface LogLabelsArrayProps {
+  labels: string[];
+}
+
+export const LogLabelsList = ({ labels }: LogLabelsArrayProps) => {
+  const styles = useStyles2(getStyles);
+  return (
+    <span className={cx([styles.logsLabels])}>
+      {labels.map((label) => (
+        <LogLabel key={label} styles={styles} tooltip={label}>
+          {label}
+        </LogLabel>
+      ))}
+    </span>
+  );
+};
+
+interface LogLabelProps {
+  styles: Record<string, string>;
+  tooltip?: string;
+  children: JSX.Element | string;
+}
+
+const LogLabel = ({ styles, tooltip, children }: LogLabelProps) => {
+  return (
+    <span className={cx([styles.logsLabel])}>
+      <span className={cx([styles.logsLabelValue])} title={tooltip}>
```

Comment:
Do we want to use the opportunity to use a "real" Grafana Tooltip rather than the `title`?
https://developers.grafana.com/ui/canary/index.html?path=/story/overlays-tooltip--basic

Reply:
![imagen](https://github.com/grafana/grafana/assets/1069378/d0860d8b-7741-49a6-b4b1-e7d97e317def)

![imagen](https://github.com/grafana/grafana/assets/1069378/2cd57727-d54e-45b4-8a2e-5078fc135442)

Much much better.